scripts/OrbitPredictor/OrbitPrediction.gd

```diff
@@ -0,0 +1,95 @@
+extends Spatial
+
+
+var orbit_animation : Animation
+var time : float = 0
+export var seemslikeorbit : bool = false
+export var AnimationPlayerchild = "AnimationPlayer"
+var GravityObjects
```

Comment:
This is array and should be initialized like array.
`
var gravity_objects := []
`

Reply:
Good idea but I think
var Gravity objects : Array
Would be even better

Reply:
every variable has some initialization like "time" line 5, so arrays and dictionaries should be initialized in similar way for consistency sake and written in snake_case not CamelCase convention.
`
var example_array := []
var example_dictionary := {}
`

Reply:
Sorry didnt notice it is a outdated file this isn't used anymore